qt_ui/windows/GameUpdateSignal.py

```diff
@@ -13,6 +14,7 @@ class GameUpdateSignal(QObject):
     instance = None
     gameupdated = Signal(Game)
     budgetupdated = Signal(Game)
+    gameover = Signal(TurnState)
```

Comment:
Oh, you _do_ have this signal. Probably better to respond to it in each component that needs to rather than in their parent. We don't want to accidentally lose behavior just because a parent changed.

Reply:
> Oh, you _do_ have this signal.

Euhm, *duh*...

> Probably better to respond to it in each component that needs to rather than in their parent.

That'll require more work, since you mentioned on discord that you don't even think this is worth solving until qt is gone, then I'm not sure what the problem is...

> We don't want to accidentally lose behavior just because a parent changed.

What behavior? If this signal is triggered, there shouldn't be any behavior since the campaign ended. All only need to assure the user can't do anything that might possibly revert back the state. From my understanding that means doing the following:

- Release the game by triggering the `gameupdated` signal, this has the following effects:
  * Disables all "game-actions" (save game, options, stats and notes, i.e. icons in the top-left corner)
  * "Disables" the map, i.e. none of the TGOs/CPs can be opened anymore
- Disable buttons in QTopPanel:
  * Air Wing, as this might create ferry flights
  * Transfers, a precaution because we released the game
  * SimSpeedControls, also a precaution
  * Pass Turn, finishing/initializing a turn when the game is "released" is definitely a problem
  * Take Off, same as Pass Turn
 
The combination of both points ought to result in a "frozen" environment, which I assume is what we want when a campaign end...